juriscraper/opinions/united_states/state/illappct.py

```diff
@@ -28,9 +33,15 @@ def _get_docket_numbers(self) -> List[str]:
         """
         dockets_numbers = []
         for case in self.cases:
-            m = re.search(self.docket_re, case["neutral_citation"])
-            raw_docket = m.group("docket")
-            district = m.group("district")
-            docket = f"{district}-{raw_docket[0:2]}-{raw_docket[2:]}"
-            dockets_numbers.append(docket)
+            match = re.search(self.docket_re, case["neutral_citation"])
+            if match:
+                raw_docket = match.group("docket")
+                district = match.group("district")
+                docket = f"{district}-{raw_docket[0:2]}-{raw_docket[2:]}"
```

Comment:
I think an example here would be helpful

Reply:
ie ... # raw_docket looks like this .... and what the final docket number looks like too.

Reply:
Examples added